tests/suite1-main.cc

```diff
@@ -188,6 +188,12 @@ main (int argc, char *argv[])
       bench_aida();
       return 0;
     }
+  if (argc >= 2 && argv[1] && std::string ("--resampler") == argv[1])
+    {
+      Bse::init_async (&argc, argv, argv[0], args);
+      extern int test_resampler (int, char **);
```

Comment:
NEVER inline prototypes, ALWAYS use a header file!
Without any exception.
I've once spent days debugging misbehaviour due to mismatching prototype, never again!

Reply:
OK, should be fixed once you merge https://github.com/tim-janik/beast/pull/126